static/app/views/starfish/views/spanSummaryPage/index.tsx

```diff
@@ -371,29 +371,24 @@ export default SpanSummaryPage;
 
 const getDescriptionLabel = (location: Location, spanMeta: SpanMeta, title?: boolean) => {
   const module = extractRoute(location);
-  if (module === 'api') {
+  const spanOp = spanMeta[SPAN_OP];
+  if (spanOp?.startsWith('http') || module === 'api') {
```

Comment:
soo yeah this is pretty strange. couple of things here

`extractRoute` is used to get the module part of the path, which doesn't map identically with the `ModuleName` enum. ie we use `api` and `database` in the url path while the enums are `http` and `db` 😢 . So its not easy to change right now because some parts of the code rely on `extractRoute` to match the path. I should have renamed the variable name from `module` to something else to help with this distinction.

Another thing is, I actually don't need to check the `module`/`extractRoute`/path/or whatever we call it, because checking the spanOp already encompasses all cases. So I just deleted the usage of `extractRoute` here.

Also, one last thing is im not really sure how much clean up we're going to do to separate DB module as a standalone module. It feels like a lot of this logic will need to be deleted anyways in the near future.